### DISCUSSION TITLE
Error sending magic link email: gomail: could not send email 1: 450 Message not queued: recipient is suppressed

### DISCUSSION THREAD

**Discussion by JulianSwales** [supabase/supabase#14916] [Questions] [answered]
We are using email OTP.. We see sometimes that a user gets HardBounced with an Email Banned or not found error... We then ask the user to get that fixed at their end.. All good.. Now however Supabase has clearly added that email to some sort of suppression list.. Which I completely understand.. Anyone know how to get a user removed from the supabase suppression list? I haven't found anywhere in the front end to do so... 

Thanks, Julian

**Comment by GaryAustin1** [comment]
I've not seen any management of email bans.  BUT you really should not be using Supabase SMTP for production...
`Use a custom SMTP server for auth emails so that your users can see that the mails are coming from a trusted domain (preferably the same domain that your app is hosted on). Grab SMTP credentials from any major email provider such as SendGrid, AWS SES, etc.`
https://supabase.com/docs/guides/platform/going-into-prod

Another comment from an employee:
"
Hey, please make sure you read our [Production Readiness](https://supabase.com/docs/going-into-prod) document. The basic SMTP sending feature is best effort only and there to get you started, but it is definitely not meant for production. You should bring your own SMTP server which is configured properly.

Email deliverability is a complex topic, and often there's no easy way to figure out why some emails are not reaching their destinations. It's always a good idea to setup DKIM, DMARC and SPF rules on your domains which significantly increases deliverability.
"
https://github.com/supabase/gotrue/issues/492#issuecomment-1225606821

**Reply by JulianSwales** [reply]
Thanks Gary.. I'm not using Supabase SMTP for production.. We are using a custom SMTP solution using mailersend.. However, since the email originates from supabase, and it's supabase giving me this message, I'm trying to understand what supabase is doing... Now it may be that it's getting some message back from mailersend, but I wasn't sure what was going on behind the scenes so to speak, so was looking for suggestions.. I'll take another look at mailersend since it may be sending back the block result, and the message I get is a result of that.. We saw the original hard bounce event in mailersend..
Thanks, Julian...

**Comment by JulianSwales** [comment, marked as answer]
So for anyone else who may run into this I discovered that the message we were getting back from Supabase is actually the message passed back from our SMTP server. That's not obvious in the message makeup.. 
So what happens with our SMTP provider (mailersend) is the email request from Supabase hard bounces when mailersend sends the email. Mailersend automatically puts that email address in a banned list. Then when supabase requests they send an email again, mailersend sends back a banned message to Supabase which turns that into a Status 500 error with the message. 
Julian.
